From 5a2998cd158d67f44195e911a538dc51db068b3e Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:48:42 -0400
Subject: [iommu] amd: Add include of <linux/irqreturn.h>

Message-id: <20140806034842.1438.37788.stgit@gir.stowe>
Patchwork-id: 87452
O-Subject: [RHEL7.1 PATCH 43/69] iommu/amd: Add include of <linux/irqreturn.h>
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 4b180d97b124a53e9fdd7fae977c856d54c1d196

commit 4b180d97b124a53e9fdd7fae977c856d54c1d196
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Feb 14 14:08:51 2014 -0700

    iommu/amd: Add include of <linux/irqreturn.h>

    We currently include <linux/irqreturn.h> in <linux/pci.h>, but I'm about to
    remove that from linux/pci.h, so add explicit includes where needed.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/iommu/amd_iommu_types.h b/drivers/iommu/amd_iommu_types.h
index 0285a21..5d9660e 100644
--- a/drivers/iommu/amd_iommu_types.h
+++ b/drivers/iommu/amd_iommu_types.h
@@ -25,6 +25,7 @@
 #include <linux/list.h>
 #include <linux/spinlock.h>
 #include <linux/pci.h>
+#include <linux/irqreturn.h>
 
 /*
  * Maximum number of IOMMUs supported
-- 
1.7.1